crates/accelerate/src/equivalence.rs

```diff
@@ -682,13 +704,13 @@ impl EquivalenceLibrary {
         }
     }
 
-    /// Retrieve the `NodeIndex` that represents a `Key`
+    /// Retrieve the [NodeIndex] that represents a [Key].
     ///
-    /// # Arguments:
-    /// * `key`: The `Key` to look for.
+    /// ## Arguments:
```

Comment:
Is there a reason to go with H2 here?

Reply:
Nothing than just me thinking H1 looks a bit too big here.

Reply:
Heh, understandable. Regardless, it looks like the Rust standard library uses H1 for a first level section, but it also looks like documenting arguments separately like we do in Python is actually not hip in Rust. 

I would recommend that we just remove the `Arguments` and `Returns` sections on all of these methods altogether. To describe arguments that aren't self-explanatory, add a second paragraph after the summary sentence to describe the usage.

Here's what the Rust standard library uses as their convention: https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text